src/config/swagger.ts

```diff
@@ -0,0 +1,29 @@
+import swaggerJsdoc from "swagger-jsdoc";
+import { version } from "../../package.json";
+
+const options: swaggerJsdoc.Options = {
+  definition: {
+    openapi: "3.0.0",
+    info: {
+      title: "Goal Tracker REST API Docs",
+      version,
+    },
+    components: {
+      securitySchemes: {
+        bearerAuth: {
+          type: "http",
+          scheme: "bearer",
+          bearerFormat: "JWT",
+        },
+      },
+    },
+    security: [
+      {
+        bearerAuth: [],
+      },
+    ],
+  },
+  apis: ["./src/routes/**/*.ts", "./src/schema/*.ts", "./src/dtos/*.ts"],
```

Comment:
We name dtos ---> dto, it is clear enough that there can be multiple

Reply:
Fixed!